build.gradle

```diff
@@ -63,8 +63,6 @@ task systemTest(type: Test, description: 'Runs the system tests.', group: 'Verif
     systemProperties System.properties.findAll { property ->
         property.key.startsWith("systemTest.")
     }
-    systemProperty 'junit.jupiter.execution.parallel.enabled', 'TRUE'
-    systemProperty 'junit.jupiter.execution.parallel.mode.default', 'CONCURRENT'
```

Comment:
I was wondering why tests stopped running in parallel. I put these values in a properties file under system tests instead. Not sure that was the right thing to do.

Reply:
It was, I think before I was getting them extended from the test task setup, wasn't sure why they stopped showing up, and guessed that maybe they were overwritten by setting systemProperties here. It works with the properties file under in this packages resources though and that is more consistent with what we are already doing so I deleted this.